.github/workflows/check-labels.yml

```
@@ -31,11 +31,7 @@ jobs:
           echo "Cloning repo with labels specs"
 
           # Temporary, before https://github.com/paritytech/labels/pull/29 is not merged
-          git clone https://github.com/paritytech/labels
-          cd labels
-          git fetch origin main
-          git checkout main
-          cd ..
+          git clone https://github.com/paritytech/labels --depth 1 --main
```

```suggestion
          git clone https://github.com/paritytech/labels --depth 1 --branch main
```
My first comment was wrong, sorry.